testbench/rest_server.py

```diff
@@ -979,6 +979,22 @@ def object_insert(bucket_name):
         blob, projection = gcs_type.object.Object.init_media(flask.request, bucket)
     elif upload_type == "multipart":
         blob, projection = gcs_type.object.Object.init_multipart(flask.request, bucket)
+        testbench.common.extract_instruction(request, context=None)
```

Comment:
This should work for both multipart and media; they are just diff flavors of single shot upload (though the Go SDK only uses multipart, but other SDKs might use media). See https://cloud.google.com/storage/docs/json_api/v1/objects/insert

Reply:
Ohh. Thanks. Done. Supported for both.